scripts/update_app_version.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+export app_version=$(git rev-parse HEAD)
```

Comment:
FWIW I don't think this would work outside of the single shell session where the script is invoked, so it wouldn't show in production.